### PR TITLE
Ad3

### PR DESCRIPTION
**Pull request by Sebastianborszcz**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,6 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not edit this file.

File: recipes/new/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "simplejson" %}
```
Comment (synapticarbors) [inline recipes/new/meta.yaml]: This file should be deleted.

File: recipes/ad3/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  license_family: GPL3
+  license_file: LICENSE
+  summary: 'AD3 Alternating Directions Dual Decomposition'
+  description: 'AD3 Alternating Directions Dual Decomposition'
+  documentation_url:  https://github.com/DerThorsten/AD3
+  dev_url: https://github.com/DerThorsten/AD3
+
+extra:
+  recipe-maintainers:
+- DerThorsten
```
Comment (synapticarbors) [inline recipes/ad3/meta.yaml]: The recipe maintainer needs to be indented. Also, you should add yourself as a recipe maintainer and ping the author if you think they might be interested in co-maintaining the recipe with you.

**Comment by synapticarbors** [comment]
Please revert all changes to `recipes/example/meta.yaml`. Pull requests should not modify the example file.

**Comment by synapticarbors** [comment]
It looks like you're having some difficulty. If you have specific questions, please ask. From looking at your commits, I think you should be specifying:

```
source:
    fn: AD3-{{ version }}.tar.gz
    https://github.com/Sebastianborszcz/AD3/archive/{{ version }}.tar.gz
```

Then I run the following to get the sha256:

```
$ shasum -a 256 ~/Downloads/AD3-2.5.3.tar.gz
1a63e2e401a33467febc22ca10e91d7385c6813b2c3b44e96460360b76db147c  /Users/X/Downloads/AD3-2.5.3.tar.gz
```

Note, I didn't see the `2.5.4` release up on Github.

**Comment by synapticarbors** [comment]
@scopatz -- This merge removed all of the comments from the example recipe. 

https://github.com/conda-forge/staged-recipes/pull/4368#pullrequestreview-74772689

**Comment by scopatz** [comment]
Whoops, you are right.  For some reason I missed that in the diff I was looking at and so I thought it had been addressed.  Reverted.

**Pull request merged**